new/src/sections/events/eventItem.tsx

```diff
@@ -0,0 +1,32 @@
+interface Props {
+  name: string;
+  details: string[];
+  description: string;
+  link: string;
+  icon: string;
+}
+
+const EventItem = ({ name, details, description, link, icon }: Props): JSX.Element => {
+  const tagline = details.join(" • ");
```

Comment:
You may want to use `useMemo` hook, so that constant calculations are not refreshed on every re-render.